pontoon/base/static/js/main.js

```diff
@@ -485,28 +481,41 @@ $(function() {
       };
     }
 
-    var index = $(this).index(),
-        current = $(this).attr('class'),
+    function getDate(el) {
+      var date = $(el).find('time').attr('datetime') || 0;
+      return new Date(date).getTime().toString();
```

Comment:
Why convert this to a string and then compare? Why not just compare dates directly?